aquarius/ddo_checker/shacl_schemas/v4/remote_4.0.0.ttl

```diff
@@ -143,6 +143,56 @@ schema:AlgorithmShape
         sh:node schema:ContainerShape ;
         sh:minCount 1;
         sh:maxCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:consumerParameters;
+        sh:node schema:ConsumerParametersShape ;
+        sh:maxCount 1;
+    ] ;.
+
+
+schema:ConsumerParametersShape
+    sh:targetClass schema:ConsumerParmeter ;
+    sh:property [
+        sh:path schema:name;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:type;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:maxCount 1;
```

Comment:
Can you help me understand why `type`, `label`, `required`, `description`, and `default` are not required?

Reply:
I created an issue to figure it out: https://github.com/oceanprotocol/docs/issues/1004

Reply:
The issue and related PR are merged: https://github.com/oceanprotocol/docs/pull/1005